src/api.ts

```diff
@@ -538,3 +538,38 @@ export interface TransactionAnalyticsRequest {
   toTimestamp?: number
   status?: TransactionAnalyticsStatus
 }
+
+export interface CreateIntegratorRequest {
+  name: string
+  integratorId: string
+  fee?: number
+  feeType?: IntegratorFeeType
+  defaultWallet?: string
+  chainWallets?: ChainWalletConfiguration
+}
+
+export interface UpdateIntegratorRequest {
```

Comment:
The integrator id will be included in url, not data field, correct?

Reply:
Yes, integratorId is included in params, not body 🙂